korman/operators/op_nodes.py

```diff
@@ -81,5 +122,21 @@ def execute(self, context):
             default = attrib.get("default", None)
             if default is not None and cached.is_simple_value:
                 cached.simple_value = default
+
+            argmap = {}
+            args = attrib.get("args", None)
+            # Load our default argument mapping
+            if args is not None:
+                if cached.attribute_type in pyAttribArgMap.keys():
+                    argmap.update(dict(zip(pyAttribArgMap[cached.attribute_type], args)))
+                else:
+                    print("Found ptAttribute type '{}' with unknown arguments: {}".format(cached.attribute_type, args))
+            # Add in/set any arguments provided by keyword
+            if not set(pyAttribArgMap[cached.attribute_type]).isdisjoint(attrib.keys()):
```

Comment:
I think the answer to my above question can be found here...

Reply:
Checking for a non-empty intersection doesn't care about order, so turning them into sets is fine.